backend/app/core/security.py

```
@@ -7,19 +7,26 @@
 - Credentials encryption/decryption
 """
 
-from datetime import datetime, timedelta, timezone
-from typing import Any
 import base64
 import json
+import logging
+import secrets
+from datetime import datetime, timedelta, timezone
+from typing import Any, Tuple
```

Ruff (0.13.3)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)